sopel/config/core_section.py

```diff
@@ -96,8 +96,41 @@ class CoreSection(StaticSection):
     channels = ListAttribute('channels')
     """List of channels for the bot to join when it connects"""
 
+    db_type = ChoiceAttribute('db_type', choices=[
+        'sqlite', 'mysql', 'postgres', 'mssql', 'oracle', 'firebird', 'sybase'], default='sqlite')
```

Comment:
I'd put `choices=[` down on this line, and move the `default` parameter down to its own line. Purely stylistic preference.